context/app/static/js/pages/Organs/hooks.js

```diff
@@ -0,0 +1,49 @@
+import useSearchData from 'js/hooks/useSearchData';
+
+const query = {
+  size: 0,
+  query: {
+    bool: {
+      must: { term: { 'entity_type.keyword': 'Dataset' } },
+    },
+  },
+  aggs: {
+    'origin_sample.mapped_organ': { terms: { field: 'origin_sample.mapped_organ.keyword', size: 10000 } },
+  },
+};
+
+function buildOrganToCountMap(aggsBuckets) {
+  return aggsBuckets.reduce((acc, { key, doc_count }) => {
+    acc[key] = doc_count;
+    return acc;
+  }, {});
+}
```

Comment:
You use `reduce` more fluently than me. I'd probably write it like this, trying to make it analogous to a dict comprehension in python, but really, whatever you're more comfortable with.

```suggestion
function buildOrganToCountMap(aggsBuckets) {
  return Object.fromEntries(
    aggsBuckets.map(
      ({key, doc_count}) => [key, doc_count]
    )
  ) 
}
```